requirements.txt

```diff
@@ -38,3 +38,4 @@ urllib3==1.26.12
 pyuwsgi==2.0.20
 Werkzeug==2.2.2
 PyYAML==6.0
+sqlparse==0.4.2
```

Comment:
Temporarily needed for application logic of product in order to classify spans. Should go away once the business logic moves to relay.